src/lib/features/instance-stats/getEdgeInstances.ts

```diff
@@ -0,0 +1,59 @@
+import type { Db } from '../../types/index.js';
+
+const TABLE = 'edge_node_presence';
+const GRACE_PERCENTAGE = 0.05;
```

Comment:
Gut feel but I don't think we should do this in the db query. The grace factor can be  calculated in billing elsewhere but if we put this here, it ties the grace value to a version of Unleash which feels weird. It's not challenging to tack this on where the bill is actually calculated